### ISSUE TITLE
Rep AirPods 3 compatibility?

### ISSUE THREAD

**Issue by BattleChickenHD** [d4rken-org/capod#191]
I was trying to connect my Fake AirPods 3 with an Airoha 1562E chipset to Capod. The app detects the headphones, but it doesent show proper battery life. The buds are stuck at 100% and the case at 0%. Is there a way to make the app compatible? There is a app called "FlyCC", that is compatible with my headphones. It shows correct battery life and features of my Headphones, like spatial audio. Is there a way to decompile the flycc app and make battery life monitoring word on Capod?

I am using a Samsung Galaxy A52s 5G

I can Add Debug logs, if they are needed.

Edit: here the link to the FlyCC app, if someone wants to decompile it: http://apkdown.198509.xyz/

The app also shows my airpods like this: ![Screenshot_20240225_213514_CAPod~2.png](https://github.com/d4rken-org/capod/assets/109387857/8241cc82-1383-41f9-9a98-47ae23c4fbad)

**Comment by d4rken** [comment]
The `AA` in the data represents the battery level of both pods, they are at 100%, does that value ever change?

**Comment by BattleChickenHD** [comment]
> The `AA` in the data represents the battery level of both pods, they are at 100%, does that value ever change?

No, the value doesent change. i just drained the right airpod a bit, it didnt change. The app thinks, they are both 100% battery, in the case getting charged and the case is apparently 0%Battery life. And thats not true. The flycc app for my replica airpods sais the exact battery life for the case and airpods

**Comment by d4rken** [comment]
The devs at https://github.com/steam3d/MagicPodsCore have figured out a way to read/write the airpods directly and also special handling for the `Airoha 1562E` replicas. I need to find time to look into that but I'm optimistic that will enable a lot of cool stuff for CAPod.

**Comment by BattleChickenHD** [comment]
> The devs at https://github.com/steam3d/MagicPodsCore have figured out a way to read/write the airpods directly and also special handling for the `Airoha 1562E` replicas. I need to find time to look into that but I'm optimistic that will enable a lot of cool stuff for CAPod.

Thats pretty cool, hope it works :) i prefer using your app, instead of flycc, cuz flycc doesent have a widget, your app has a very good-looking widget. Only thing i would've change is: the icon for airpods 3 is still the airpods 2 icon. If you can change it, it would look better 😄
